src/commands/Minion/open.ts

```diff
@@ -55,10 +65,14 @@ export default class extends BotCommand {
 
 		const image = await task!.generateBankImage(
 			loot,
-			`You opened a ${clueTier.name} clue and received...`
+			`You opened a ${clueTier.name} clue ${hadMimic ? 'with a mimic ' : ''}and received...`
 		);
 
 		msg.author.incrementClueScore(clueTier.id);
+		const MIMIC_ID = 23184;
+		if (hadMimic) {
+			msg.author.incrementClueScore(MIMIC_ID);
+		}
```

Comment:
this should be removed, as mimics arent clues

Reply:
if i were to check my clue kcs i would want mimic to be shown

Reply:
i would put this in monster scores but then it would need to be a killable monster which is even more wrong than it being a clue